packages/react-native-navigation/src/rum/instrumentation/DdRumReactNativeNavigationTracking.tsx

```diff
@@ -8,19 +8,26 @@ import React from 'react'
 import { ComponentDidAppearEvent, Navigation } from 'react-native-navigation';
 import { DdRum } from '@datadog/mobile-react-native';
 
+
+
+
+export type ViewNamePredicate = (trackedView: any, trackedName: string) => string | null
```

Comment:
If we want users to allow `import type { ViewNamePredicate } from '@datadog/mobile-react-native-navigation'` in their code, this type needs to be re-exported in the `index` file. Otherwise they either have to use no-type lambda with `startTracking(...)` code or specify full path like `from '@datadog/mobile-react-native-navigation/src/...'`